dune

```
@@ -0,0 +1 @@
+(env (_ (flags -bin-annot -safe-string))) ; Use the same flags as with ocamlbuild
```

side note: `-bin-annot` is optional (as it is always added to the flags) but it reflects ocamlbuild's `_tags` file more strictly if we add it by hand. But it's up to you, whatever you prefer.

---

We don't need to reflect ocamlbuild's `_tags` file the default build is `Makefile` based. We just need to reflect the standard OCaml flags.

---

The standard OCaml *warning* flags.

---

I don't really understand the request. The standard warning flag in the one used when the ocamlc/ocamlopt are called without setting `-w` so this is exactly what this is doing as it replaces the set of standard flags that dune has as its default to something that **does not** have `-w`

---

> We don't need to reflect ocamlbuild's `_tags` file the default build is `Makefile` based. We just need to reflect the standard OCaml flags.

in that case to match the flags set in `build.ml` we need:
```suggestion
(env (_ (flags -g -bin-annot -safe-string))) ; Use the same flags as with build.ml
```